src/contexts/NotificationProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
```

Comment:
Really minor but can you create a type here, say, `type NotificationVariantType = 'success' | 'danger' | 'info';` at the top, and then have all the variant params in this file be of that type? Just is a bit more specific than string since there are only a few options. Also helps cut down on repetition

Reply:
Yeah, that's a good idea.  I explicitly listed the values for the type of something.  Not sure why I didn't do it everywhere.